pyqtgraph/graphicsItems/ViewBox/ViewBox.py

```diff
@@ -574,6 +574,14 @@ def setRange(self, rect=None, xRange=None, yRange=None, padding=None, update=Tru
                     dy = 1
                 mn -= dy*0.5
                 mx += dy*0.5
+            # Make sure that the range includes a usable number of quantization steps:
+            #    approx. eps  : 3e-16
+            #    * min. steps : 10
+            #    * mean value : (mn+mx)*0.5 
+            quantization_limit = (mn+mx) * 1.5e-15 # +/-10 discrete steps of double resolution
+            if mx-mn < 2*quantization_limit:
+                mn -= quantization_limit
+                mx += quantization_limit
```

Comment:
It looks good to me. Thanks. I just have a very minor point: regarding the comment `approx. eps`, stands for approximated double resolution ?

Reply:
Yes, numpy seems to call it that. I think C spells it out as epsilon, typically.
https://numpy.org/doc/stable/reference/generated/numpy.finfo.html
According to that page: `eps = 2**-52`, approximately 2.22e-16.
But there is no real benefit to being ultra-accurate here; I am just trying to explain the magic cutoff value.